docs/src/charmil_factory.md

```diff
@@ -3,38 +3,49 @@ title: Charmil Factory
 slug: /charmil_factory
 ---
 
-The Charmil Factory offers you one-stop access to all of the Charmil packages all at once. So you just need to initialize factory in your cobra command and start using logging, i18n, etc. 
+The Charmil Factory offers you one-stop access to all of the Charmil packages all at once. So you just need to initialize factory in your cobra command and start using logging, i18n, etc.
 ```go
-import github.com/aerogear/charmil/core/factory
+import (
+    "<module-name>/<cli-name>/pkg/factory"
+    "github.com/aerogear/charmil/core/config"
+    "github.com/aerogear/charmil/core/utils/localize"
+)
 ```
 
 ## How to use
 
-1. Create a cobra command for your plugin
+1. Initialize factory in your command, by providing it an instance of charmil localizer and config handler.
 ```go
-func MyCommand() (*cobra.Command, error) {
-	cmd := &cobra.Command{
-		Use:   "my",
-		Short: "This is my command",
-		RunE: func(cmd *cobra.Command, args []string) error {
-			return nil
-		},
-	}
-
-	return cmd, nil
+// Embed provides access to files embedded in the running Go program meaning you can get your locales directory with the path to it.
+//go:embed locales/*
+defaultLocales embed.FS
+
+// create a config handler
+h := config.NewHandler("./config.json", cfg)
+
+// Loads config values from the local config file
+err := h.Load()
+if err != nil {
+    log.Fatal(err)
 }
```

Comment:
```suggestion
```
This part can be removed as it's independent and has nothing to do with the factory initialization.